cypress.config.ts

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+  e2e: {
+    baseUrl: 'https://testify.team'
+  },
+  viewportWidth: 1200,
+  retries: {
+      runMode: 3
+  }  
```

Comment:
add this 2 lines

viewportWidth: 1200,
experimentalMemoryManagement: true


Viewport default setting will make mors stability for different test systems and momory management should have power ;)